incentivizing_rewarding_participants.adoc

```diff
@@ -46,25 +46,44 @@ What they reward—either intentionally or unintentionally—is what they'll be
 
-But leaderboard systems might also be _demotivating_ for participants.
+But leaderboard systems in particular might also be _demotivating_ for participants.
 "Contributors who enjoy competition and like to engage in activities counted for the leaderboard's scoring, the desire to be on the leaderboard can inspire increased participation," https://opensource.com/article/21/9/community-leaderboard[writes community metrics expert Georg Link]. "However, contributors who participate in ways that don't get counted in the scoring would not have any chance of showing up in the leaderboard and might be discouraged by it."
 Link explains that leaderboard systems only reward the activities project maintainers choose to reward—an implicit statement about which types of contribution it deems more valuable than others.
 And all potential contributors have the time, talent, or resources to contribute in precisely those ways.
 "Consequently," Link says, "leaderboards can worsen the issues open source faces regarding creating a welcoming and inclusive environment [as] only a specific subset of contributors can be rewarded by leaderboards, regardless of how well they are designed."
 
+=== Badge systems
+
+Closely related to point systems and leaderboards are community badge systems.
+To reward participants for their contributions to an open source project, a community might create unique digital tokens or "badges" contributors receive when they've completed particular activities (see the https://badges.fedoraproject.org/[Fedora Project] for a popular example of this reward model).
+Such activities might include attending a community event, voting in a community election, updating project documentation, or assisting a new member with a request.
+Contributors can then display these badges on their project profiles as indicators of their diverse and ongoing contributions to a project.
+
+Indeed, this reward model is useful when a community seeks to incentivize a diversity of contributions.
+Unlike point systems, which might reward people for repeatedly performing the same tasks, badge systems encourage people to contribute to a project in *multiple and different ways* by rewarding them with something unique every time they attempt and accomplish something they haven't before.
+In this way, they encourage contributors to assist with several aspects of a project.
+Like point systems and leaderboards, then, badges are a useful reward model in cases where participants seem motivated by a desire to expand their reputations in a community.
+
+Badge systems can complicate community engagement efforts, however, when they are easy to manipulate or "game."
+As Matt Snell notes as part of https://opensource.ieee.org/community-advisory-group/community-badging/about/-/issues/26[recent work in this area], badge systems might incentivize participants to make only the minimum viable contribution necessary to receive a badge.
```

Comment:
Thank you for including me! I'm going by Matt Cantu Snell now 😁 that's my only remaining request